sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_client_async.py

```diff
@@ -21,13 +22,16 @@
 from .._common.utils import generate_dead_letter_entity_name, strip_protocol_from_uri
 from .._common.constants import (
     ServiceBusSubQueue,
-    ServiceBusReceiveMode
+    ServiceBusReceiveMode,
+    NEXT_AVAILABLE_SESSION,
 )
 from ._async_utils import create_authentication
 
 if TYPE_CHECKING:
     from azure.core.credentials_async import AsyncTokenCredential
 
+NextAvailableSessionType = Literal[NEXT_AVAILABLE_SESSION]  # type: ignore
```

Comment:
This line (without the type: ignore) also resulted in a mypy error for all python versions: `error: Variable "azure.servicebus.aio._servicebus_client_async.NextAvailableSessionType" is not valid as a type`, so I ignored type.
I'm not sure if there's a better alternative? @annatisch

Reply:
Hah - if we have to do `type: ignore` then the whole thing is moot and we should just remove it ;)
Did you check the `Literal` docs? My suggestion was not researched, so may have been invalid...

Reply:
yep that makes sense! I tried this method instead here: 

I had to update this line to the specific enum (`ServiceBusSessionFilter.NEXT_AVAILABLE`), to get rid of the mypy error. I think it only takes the exact enum as a parameter.

This would expose the `ServiceBusSessionFilter` to users, which I know we didn't want to do. Since we also don't want type: ignore, I'm not sure that this would work.

Thoughts @annatisch ?

Reply:
resolution: expose ServiceBusSessionFilter